src/components/AccountCircleButton.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { StyleProp, StyleSheet, View, ViewStyle } from 'react-native'
+import Touchable from 'src/components/Touchable'
+import AccountCircle from 'src/icons/AccountCircle'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { Spacing } from 'src/styles/styles'
+
+interface Props {
+  style?: StyleProp<ViewStyle>
+  size?: number
+  testID?: string
+}
+
+export default function AccountCircleButton({ style, size, testID }: Props) {
+  const onPress = () => {
+    navigate(Screens.ProfileMenu)
+  }
+
+  return (
+    <View style={styles.container}>
+      <Touchable
```

Comment:
Touchable used here and in other added icons instead of `TopBarButton.tsx` to better handle ripple on Android.

Reply:
is this something TopBarIconButton can support? if not, is there a purpose of keeping TopBarIconButton around?